third_party/terraform/utils/test_utils.go

```diff
@@ -83,6 +83,10 @@ func (d *ResourceDiffMock) Clear(key string) error {
 	return nil
 }
 
+func (d *ResourceDiffMock) ForceNew(key string) error {
+	return nil
```

Comment:
If you add a `ForceNew bool` to the `ResourceDiffMock` struct, you can use this func to set it to `true`, which lets you look up in the test whether or not the diff had been marked as ForceNew.